questions.md

```diff
@@ -0,0 +1,135 @@
+# Questions
```

Comment:
I'd remove this file from the PR.
These kind of TODO files are more a personal thing and there is no need to keep it in the repo's history for everyone to see.

Reply:
Yup, sorry, residual file, didn't mean to add it in the first place, removed.